resources/[race]/race/modes/deadline.lua

```diff
@@ -171,7 +171,7 @@ function Deadline:launch()
 	RaceMode.launch(self)
 
 	--if math.random(2) == 1 then clientCall(g_Root, 'showOnlyHealthBar', true) end -- Use this to hide names 50% of the time
-	clientCall(g_Root, 'showOnlyHealthBar', true) -- Use this to hide names 100% of the time
+	--clientCall(g_Root, 'showOnlyHealthBar', true) -- Use this to hide names 100% of the time
```

Comment:
I think the line should be removed entirely instead of being commented out, we can always revert this commit.